release-1.8/release-1.8.md

```diff
@@ -0,0 +1,79 @@
+# Proposed timeline for 1.8
+We will begin the 1.8 release the first week of July 2017.
+
+The proposal below is similar in layout to the 1.7 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+
+## 1.8 Release Schedule
+- July 5 (Weds) coding start (7w)
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1 (Tues): Feature Freeze
+- Aug 9 (Weds): v1.8.0-alpha.3
+- Aug 23 (Weds): v1.8.0-alpha.4
+- Sept 1 (Fri): Code Freeze 
+- Sept 6 (Weds): v1.8.0-beta.0, v1.9.0-alpha.0
+- Sept 13 (Weds): v1.8.0-beta.1
+- Sept 18th (Mon): v1.9.0-alpha.1
+- Sept 20th (Weds): v1.8.release-candidate
+- Sept 27th (Weds):  v1.8.0 (release!)
+
+## 1.8 Details
+
+### July 5 - Sept 1
+- 8 week coding period
+- Release 1.8 alphas every 2 weeks
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1st (Tues): Feature Freeze: all features going into release should
+  have an associated issue in the features repo
+- Aug 9th (Weds): v1.8.0-alpha.3
+- Aug 23rd (Weds): v1.8.0-alpha.4
+
+### Sept 1 (Fri) - Sept 14 (Thurs)
+- Sept 1st Begin Code Freeze
+  * Hard deadline for feature-related PRs to be in submit queue
+  * Add milestone restriction on submit queue.
+  * Community Feature Burndown Meetings held two or three times until release week (then every day). For those interested in joining please join [Kubernetes Milestone Burndown Group](https://groups.google.com/forum/#!forum/kubernetes-milestone-burndown)
+  * Focus on bugfix, test flakes and stabilization
+  * Ensure docs and release notes are written
+  * Identify all features going into the release, and make sure alpha, beta, ga is marked in features repo
+- Sept 6th (Weds): v1.8.0-beta.0
+  * Branch Manager to create v1.8 release branch.
+  * v1.9.0-alpha.0 is cut
+  * Test-infra lead to set up CI for branch.
+  * Begin regular fast-forwards (at least daily to keep CI up-to-date).
+- Sept 13th (Weds): End of Code Slush
+  * v1.8.0-beta.1 cut
+  * Final fast-forward of release branch.
+  * After this, all changes for the release must now be cherrypicked in batches by branch
+  manager.
+  * Remove milestone restriction on submit queue. Bot drains backlog over the
+  weekend.
+
+### Sept 15 - Sept 27
+- Sept 18th (Mon): v1.9.0-alpha.1
+- Sept 20th (Weds): v1.8.release-candidate
+  * RC means no known release-blockers outstanding.
+  * Only accept cherrypicks for release-blockers.
+  * Announce on mailing lists, Twitter, etc. to ask people to build and test the RC and submit feedback
+  * Perhaps managed k8s providers make rc.1 available on early access channels.
+  * More RCs as needed to verify release-blocking fixes.
```

Comment:
By now, we're 1 week out from release.  From a release notes perspective, I'm in full agreement we should basically have everything documented, all we should be changing by now are structural edits or last minute Known Issues.  We should have enough in place that the release branch manager could cut a copy of CHANGELOG.md with all the release notes incorporated (assuming that's where they still land)

IMO what should _not_ be happening by now is the addition of features or large structural changes in the release notes.  I feel like I saw both happen the final week of the previous release.

From a docs perspective, I would hope we're basically at a live copy of all of the docs, with grammar edits and spell checking at this point.  Historically that has not been the case though.  What is reasonable to expect by now? @kubernetes/sig-docs-maintainers